metrics-operator/api/v1alpha3/keptnmetricsprovider_types.go

```diff
@@ -36,9 +36,7 @@ type KeptnMetricsProviderSpec struct {
 	SecretKeyRef corev1.SecretKeySelector `json:"secretKeyRef,omitempty"`
 }
 
-// KeptnMetricsProviderStatus defines the observed state of KeptnMetricsProvider
-type KeptnMetricsProviderStatus struct {
-}
+type EmptyStatus string
```

Comment:
Should we make this an empty `struct` instead, as this is also what kubebuilder does by default?